modules/systemdstates/systemdstates.go

```diff
@@ -0,0 +1,79 @@
+package systemdstates
+
+import (
+	"github.com/netdata/go-orchestrator/module"
+	"github.com/netdata/go.d.plugin/pkg/matcher"
+)
+
+type Config struct {
+	Selector matcher.SimpleExpr `yaml:"selector"`
+}
+
+func init() {
+	creator := module.Creator{
+		Defaults: module.Defaults{
+			Disabled:    false,
+			UpdateEvery: 1,
+		},
+		Create: func() module.Module { return New() },
+	}
+
+	module.Register("systemdstates", creator)
+}
+
+// New creates SystemdStates with default values
+func New() *SystemdStates {
+	return &SystemdStates{
+		charts:  charts.Copy(),
+		metrics: make(map[string]int64),
+	}
+}
+
+// SystemdStates SystemdStates module
+type SystemdStates struct {
+	module.Base  // should be embedded by every module
+	Config       `yaml:",inline"`
+	metrics      map[string]int64
+	charts       *module.Charts
+	unitsMatcher matcher.Matcher
```

Comment:
Lets call it `selector` or (`sr`). Configuration option has `selector` name too.

`metrics      map[string]int64` field is not needed (it is not used)